### PR TITLE
HAWQ-289. Fix faultinjector in client and server for hawq

### PR DESCRIPTION
**Pull request by realdawn**

### CODE REVIEW

File: tools/bin/hawqpylib/hawqarray.py [hunk 1]
```diff
@@ -514,15 +514,14 @@ def initFromCatalog(dbURL, utility=False, useAllSegmentFileSpaces=False):
...
                                       CASE
                                           WHEN sc.registration_order <= 0 THEN '%s'
                                           ELSE '%s'
                                       END AS datadir
-                               FROM pg_catalog.gp_segment_configuration sc,
-                                    pg_catalog.pg_filespace fs,
-                                    pg_catalog.pg_filespace_entry fse
-                               WHERE fse.fsefsoid = fs.oid
+                               FROM pg_catalog.gp_segment_configuration sc
+
```
Comment (huor) [inline tools/bin/hawqpylib/hawqarray.py]: The blank lines here need to be removed.

File: src/backend/utils/misc/faultinjector.c [hunk 2]
```diff
@@ -1001,7 +1002,7 @@ FaultInjector_NewHashEntry(
 	{
 		case ChangeTrackingDisable:
 		case FileRepConsumerVerification:
-		case StartPrepareTx:
+
```
Comment (huor) [inline src/backend/utils/misc/faultinjector.c]: The blank line here needs to be removed.

**Comment by huor** [comment]
The overall fix looks good. +1

**Comment by liming01** [comment]
Looks good. +1

**Pull request closed without merging**